wsd/DocumentBroker.cpp

```diff
@@ -1934,10 +1941,8 @@ size_t DocumentBroker::removeSession(const std::string& id)
         }
         std::shared_ptr<ClientSession> session = it->second;
 
-        // Last view going away, can destroy.
-        if (_sessions.size() <= 1)
```

Comment:
Ensuring we handle _sessions.size == 0 carefully is important to not leak sessions of course.

Reply:
Restored.

Reply:
Not sure about the previous version of the patch; the current one turned <= 1 into == 1; but as we talked with @Ashod , we cannot get this far with empty _sessions (we search the id in there, and return if it was not found).